doc/lsst.meas.algorithms/refCatCreation.rst

```diff
@@ -0,0 +1,145 @@
+#########################################
+How to generate an LSST reference catalog
+#########################################
+
+The process for generating an LSST-style HTM indexed reference catalog is similar to that of running other LSST Tasks: write an appropriate ``Config`` and run :py:class:`~lsst.meas.algorithms.IngestIndexedReferenceTask`.
+The differences are in how you prepare the input data, what goes into that ``Config``, how you go about running the ``Task``, and what you do with the final output.
+
+Ingesting a large reference catalog can be a slow process.
+It took about 24 hours to ingest all of Gaia DR2 on an Intel i7-6700K (8 processes) with a 3-disk RAID 10 array.
```

Comment:
I see your point, but I would be concerned that a more generic statement like that would become out of date and lose context. How about this, since the parallelization is a major aspect to the runtime? "The Gaia DR2 took about 24 hours to ingest on a high-performance workstation running with 8 parallel processes, for example."

Reply:
Sure. I just want to make sure that it doesn't sound like it's trying to be a benchmark or a similar super-specific measurement. Just a guideline that this task takes on the order of a day to run.